charts/openstad-headless/templates/image/deployment.yaml

```diff
@@ -33,6 +33,8 @@ spec:
             value: "31451"
           - name: IMAGES_DIR
             value: {{ .Values.image.volumes.data.mountPath | default "/opt/openstad-headless/apps/image-server/images" }}
+          - name: DOCUMENTS_DIR
+            value: {{ .Values.image.volumes.data.mountPath | default "/opt/openstad-headless/apps/image-server/documents" }}
```

Comment:
Ik denk dat je hier een nieuwe value voor aan moet maken, `documentMountPath` bijv. want iets verderop in dit bestand heb je ook de `volumeMounts`, waarin nu een volume gemount wordt op de `.Values.images.volumes.data.mountPath`, met als fallback de images folder.

```
        volumeMounts:
          - mountPath: {{ .Values.image.volumes.data.mountPath | default "/opt/openstad-headless/apps/image-server/images" }}
            name: data-vol
```

Volgens mij moet daar een extra volume bij voor de documents, met het juiste fallback path.

Reply:
Kun je dit opnieuw controleren of het zo goed staat?